### PR TITLE
framework/eventloop: Fix wrong word in doxygen and typo

### PR DESCRIPTION
**Pull request by jeongarmy**
Fix wrong word in doxygen and typo in source file.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [5961145fd536d666af00d918de2dfd94215bd707] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [5961145fd536d666af00d918de2dfd94215bd707] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [c99acb4d9ea84726a96e4e85432c0871f417d683] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c99acb4d9ea84726a96e4e85432c0871f417d683] -

File: framework/include/eventloop/eventloop.h [hunk 1]
```diff
@@ -71,8 +71,8 @@ enum el_event_type_e {
  * @brief Return value of callback function for event
  * @details The callback function should return EVENTLOOP_CALLBACK_STOP(false) or EVENTLOOP_CALLBACK_CONTINUE(true).\n
  * It decides whether continuing to execute callback function for the event or not.\n
- * If it returns EVENTLOOP_CALLBACK_STOP, receving registered events is stopped.\n
- * The registered handler won't be executed from now on although receving the event.\n
+ * If it returns EVENTLOOP_CALLBACK_STOP, receiving registered events is stopped.\n
+ * The registered handler won't be executed from now on although receiving the event.\n
  * Otherwise, a callback funtion is called continuously.
```
Comment (sunghan-chang) [inline framework/include/eventloop/eventloop.h]: function

**Comment by seinfra** [comment]
Target : [16000c2d5cdfc072df11b5ce6c1a768d0924afe0] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [16000c2d5cdfc072df11b5ce6c1a768d0924afe0] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6a0dcc5e11fd0f3884205c7f9640bbc777ac1722] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [6a0dcc5e11fd0f3884205c7f9640bbc777ac1722] - Code Rule Check (C++) OK.

**Pull request merged**